test/DebuggerCommon/step_in_from_interpreted_function_attach.js.dbg.baseline

```diff
@@ -2,19 +2,19 @@
   {
     "callStack": [
       {
-        "line": 27,
+        "line": 29,

```

Comment:
If WScript.Platform is not needed for majority of the cases then exposing it under a switch is ideal otherwise update the core baseline without modifying the test.